components/Menu/Menu.js

```diff
@@ -55,6 +56,12 @@ export default function Menu({
     }
   }, [isCollectionsClosing]);
 
+  function getNumberOfCardsInCollection(id) {
+    const cardsInCollection = getAllFlashcardsFromCollection(id);
+    const numberOfCards = cardsInCollection.length;
+    return numberOfCards;
+  }
+
```

Comment:
If the calculation is only needed once, it often makes sense to perform it directly in the return statement. This makes the code shorter, and keeps the logic in one place.
You can immediately see what’s happening without having to split the code across different functions. See my suggestion below.

If you need the number of cards in multiple places in the code, it makes sense to encapsulate the logic in its own function, of course. 🙂